pkg/apis/core/v1beta1/helper/shoot.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strconv"
 
 	"github.com/Masterminds/semver/v3"
+	"golang.org/x/exp/maps"
```

Comment:
`golang.org/x/exp/maps.Clone` under the hood invokes `maps.Clone`. Why don't we use `maps.Clone` directly?

```suggestion
	"maps"
```